sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -274,12 +275,12 @@
  *     .fromQuery("SELECT year, mean_temp FROM [samples.weather_stations]"));
  * }</pre>
  *
- * <p>Users can optionally specify a query priority using {@link TypedRead#withQueryPriority(
- * TypedRead.QueryPriority)} and a geographic location where the query will be executed using {@link
- * TypedRead#withQueryLocation(String)}. Query location must be specified for jobs that are not
- * executed in US or EU, or if you are reading from an authorized view. See <a
- * href="https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query">BigQuery Jobs:
- * query</a>.
+ * <p>Users can optionally specify a query priority using {@link
+ * TypedRead#withQueryPriority(TypedRead.QueryPriority)} and a geographic location where the query
+ * will be executed using {@link TypedRead#withQueryLocation(String)}. Query location must be
+ * specified for jobs that are not executed in US or EU, or if you are reading from an authorized
```

Comment:
Did you mean to mention "withQueryTempDataset" here ?

Reply:
No, I accidentally reformatted these lines, that's why they are in the PR. Let me see if I can undo these changes.